test/PrintAsObjC/swift_name.swift

```diff
@@ -31,3 +31,13 @@ class Test : NSObject {
   func useAlignment(_: ZZAlignment) {}
   func useObjects(_: ZZClass) -> [ZZProto] { return [] }
 }
+
+@objc
+public enum TestE : Int{
+	@objc(A2)
+	case A1
+	case B1
+}
+// CHECK: typedef SWIFT_ENUM(NSInteger, TestE)
+// CHECK-NEXT:  A2 SWIFT_COMPILE_NAME("A1") = 0,
```

Comment:
I put the `{{^}}` in my example deliberately; this CHECK line will match the previous version as well.